### PR TITLE
Feature: Admissions area of study adjustments

### PR DESCRIPTION
**Pull request by bspeare**
# In this pull request:

- Moved Admissions Requirements text back into main column
- Added custom headline block above Admission Requirements next steps
- Fixed equal height cards for requirements
- Fixed stats to display inline

## How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`

Stat on http://admissions.local.drupal.uiowa.edu/areas-of-study/accounting should look like:

Screen Shot 2021-02-09 at 12 20 21 PM

Requirements Layout on http://admissions.local.drupal.uiowa.edu/areas-of-study/accounting should look like:

Screen Shot 2021-02-09 at 12 20 12 PM

### CODE REVIEW

File: docroot/sites/admissions.uiowa.edu/modules/admissions_core/src/Plugin/Block/AOSRequirements.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  public function defaultConfiguration() {
+    return ['label_display' => FALSE];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $markup = '<div class="element--bold-intro text-align-center">
+            New Headline
```
Comment (richardbporter) [inline docroot/sites/admissions.uiowa.edu/modules/admissions_core/src/Plugin/Block/AOSRequirements.php:30]: Is this supposed to say New Headline?

Comment (bspeare) [reply docroot/sites/admissions.uiowa.edu/modules/admissions_core/src/Plugin/Block/AOSRequirements.php:30]: Yes, they don't know what they want it to say yet.

**Review by richardbporter — APPROVED** [review]
Looks good. I need to add a field after this gets merged. A follow up task might be to run that yet-to-be-determined text through translation like: 
```
      '#markup' => $this->t('New Headline'),
      '#prefix' => '<div class="element--bold-intro text-align-center">',
      '#suffix' => '</div>',
``` 
or something similar.

**Comment by bspeare** [comment]
Thanks @richardbporter I'll try to check-in with you when they decide on a headline.

**Pull request merged**